oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java

```diff
@@ -35,88 +36,107 @@
     public static final String ENDPOINT_NAME = "endpoint_name";
     public static final String ENDPOINT_ID = "endpoint_id";
     public static final String TRACE_ID = "trace_id";
+    public static final String TRACE_SEGMENT_ID = "trace_segment_id";
+    public static final String SPAN_ID = "span_id";
     public static final String IS_ERROR = "is_error";
-    public static final String STATUS_CODE = "status_code";
     public static final String CONTENT_TYPE = "content_type";
     public static final String CONTENT = "content";
     public static final String TIMESTAMP = "timestamp";
+    public static final String TAGS = "tags";
 
     @Setter
     @Getter
     @Column(columnName = SERVICE_ID)
-    private int serviceId;
+    private String serviceId;
     @Setter
     @Getter
     @Column(columnName = SERVICE_INSTANCE_ID)
-    private int serviceInstanceId;
+    private String serviceInstanceId;
     @Setter
     @Getter
     @Column(columnName = ENDPOINT_ID)
     private String endpointId;
     @Setter
     @Getter
-    @Column(columnName = ENDPOINT_NAME)
+    @Column(columnName = ENDPOINT_NAME, matchQuery = true)
     private String endpointName;
     @Setter
     @Getter
-    @Column(columnName = TRACE_ID)
+    @Column(columnName = TRACE_ID, length = 150)
     private String traceId;
     @Setter
     @Getter
-    @Column(columnName = IS_ERROR)
-    private int isError;
+    @Column(columnName = TRACE_SEGMENT_ID, length = 150)
+    private String traceSegmentId;
+    @Setter
+    @Getter
+    @Column(columnName = SPAN_ID)
+    private int spanId;
     @Setter
     @Getter
-    @Column(columnName = STATUS_CODE)
-    private String statusCode;
+    @Column(columnName = IS_ERROR)
+    private int isError;
     @Setter
     @Getter
-    @Column(columnName = CONTENT_TYPE)
+    @Column(columnName = CONTENT_TYPE, storageOnly = true)
     private int contentType = ContentType.NONE.value();
     @Setter
     @Getter
-    @Column(columnName = CONTENT)
+    @Column(columnName = CONTENT, length = 1_000_000, storageOnly = true)
```

Comment:
If this `storageOnly = true`, we should consider `binary` rather than `string`. 
At the same time, depend on your suggestions here, https://github.com/apache/skywalking-query-protocol/pull/47